tests/unit_tests/test_wave_03.py

```diff
@@ -130,8 +130,11 @@ def test_swap_items_from_their_empty_returns_false():
     nobodys_item = Item()
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
-
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result == False
+    assert nobodys_item not in fatimah.inventory
+    assert nobodys_item not in jolie.inventory
```

Comment:
The feedback in wave 1 applies here as well, what else would be helpful to assert to ensure there were no unintended side effects?

Reply:
assert fatimah.inventory == [item_a, item_b, item_c]
    assert jolie.inventory == []
    assert item_a in fatimah.inventory
    assert item_b in fatimah.inventory
    assert item_c in fatimah.inventory

Added above assertions in the test . Thank you.